app/components/Riddle.jsx

```diff
@@ -0,0 +1,46 @@
+import React, {Component} from 'react';
+
+import {GridList, GridTile} from 'material-ui/GridList';
+import {List, ListItem} from 'material-ui/List';
+import Divider from 'material-ui/Divider';
+import Subheader from 'material-ui/Subheader';
+import Avatar from 'material-ui/Avatar';
+import {grey400, darkBlack, lightBlack} from 'material-ui/styles/colors';
+import {connect} from 'react-redux';
+import Paper from 'material-ui/Paper';
+import IconMenu from 'material-ui/IconMenu';
+import MenuItem from 'material-ui/MenuItem';
+//import FontIcon from 'material-ui/FontIcon';
+import IconButton from 'material-ui/IconButton';
+import AddPhoto from 'material-ui/svg-icons/image/add-a-photo';
+import AddImage from './AddImage';
+
+export default class Riddle extends Component {
```

Comment:
This should be a presentational (ie functional component) as nothing is being held on local state. You'll also need to change the props in this case from `this.props` to just be `props`